database/sql/src/main/scala/cromwell/database/slick/tables/WorkflowMetadataSummaryEntryComponent.scala

```diff
@@ -71,11 +71,10 @@ trait WorkflowMetadataSummaryEntryComponent {
   val workflowMetadataSummaryEntryIdsAutoInc = workflowMetadataSummaryEntries returning
     workflowMetadataSummaryEntries.map(_.workflowMetadataSummaryEntryId)
 
-  val rootWorkflowIdsByArchiveStatusAndEndedOnOrBeforeThresholdTimestamp = Compiled(
+  val workflowIdsByArchiveStatusAndEndedOnOrBeforeThresholdTimestamp = Compiled(
```

Comment:
Although yours definitely sounds nicer, I think it's more _technically_ correct as it is, because it returns workflow IDs and you can choose which archive status to filter by.